Corona-Warn-App/src/main/assets/consent_de.html

```diff
@@ -0,0 +1,15 @@
+<p>
+    Einwilligungserklärung
```

Comment:
That headline is part of a string (`interop_consent_headline`). So the lines 1-3 should be removed, also in the other languages.